### ISSUE TITLE
Finish developing complex image object ingest & display

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#24]

**Comment by rmjaffe** [comment]
-[ ] Ingest Porter College Collection into staging site
Make sure child images display in correct order
Configure parent pages tooad universal viewer by default
Load child metadata manifest into Universal Viewer "metadata" sidebar using AJAX
Hide links to child URLs & forward requests to those URLS
   We decided that some child works will not have their own display pages, but will only be displayed as part of the parent work. 
   These child work urls will point to the parent work page with the child work highlighted.

Check and fix hidden work indexing
   When a child work does not display on its own page but has unique metadata, we should ensure that this metadata is indexed (or not) the way we want. If you search for metadata of a hidden child work (e.g. a child work called "table of contents" that does not display alone) should you get the parent work (e.g. books that have tables of contents) or nothing? I developed this flexibly but configured it haphazardly, so I need to write down exactly how it works now so I can ask exactly how we want it to work. Reconfiguration of the details should be easy once the software works. 

Check and fix metadata inheritance
   A lot of times, child works share most of their metadata with their parent works. We have come up with a system in which we can specify that the child work inherits certain metadata fields, so we don't have to enter them for every child work in a series. This is mostly working but it needs test coverage, debugging, and some actual works to test it on. 

Deploy these changes to production site
Ingest Collections of Complex Objects
Ingest Porter College Collection into production site
Ask whether to initiate ingest of complex objects from campus history collection
